### PR TITLE
Add psalm integration

### PR DESCRIPTION
**Pull request by kea**
Signed-off-by: Manuel Kea Baldassarri <kea@keasoft.it>

|    Q          |   A
|-------------- | ------
| QA            | yes

### Description

As requested on #1, I've integrated psalm.

### CODE REVIEW

**Review by Ocramius — CHANGES_REQUESTED** [review]
We need following before merging: 

 * rebase
 * get this to green

File: psalm-baseline.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<files psalm-version="4.10.0@916b098b008f6de4543892b1e0651c1c3b92cbfa">
```
Comment (Ocramius) [inline psalm-baseline.xml]: `4.19` is in `composer.lock`, but `4.10.0` is declared here? :thinking:

File: psalm.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<psalm
+        errorLevel="3"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="https://getpsalm.org/schema/config"
+        xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
```
Comment (Ocramius) [inline psalm.xml]: some alignment wonkiness happening here :D

**Review by Ocramius — APPROVED** [review]
Some tiny adjustments - feel free to dismiss, if you don't deem them necessary.

Overall looking good, thanks @kea!

File: psalm-baseline.xml (hunk 1 above)
Comment (kea) [reply psalm-baseline.xml]: You're right! :)
I'll fix it.

**Comment by kea** [comment]
Fixed missing reference to some Zend\Expressive missing class. Now psalm check is green.

**Review by Ocramius — APPROVED** [review]
Thanks @kea!

**Pull request merged**